automated install/basic-install.sh

```diff
@@ -968,7 +979,7 @@ fi
 
 echo ":::"
 echo "::: The install log is located at: /etc/pihole/install.log"
-echo "::: View the web interface at http://pi.hole/admin or http://${IPv4addr%/*}/admin"
+echo "::: View the web interface at http://pi.hole/admin or http://${IPv4_address%/*}/admin"
 }
 
 main "$@"
```

Comment:
Warner had something he was working on with an undocumented flag that would pull the variables from a file and run non-interactive. But I understand what you mean now. Instead of triggering `main()` on load with the flag set it would allow for everything to load up and prep for main but then you could unit/nosetest the functions... I'll look at that and see how to get that for you.